source/client/scripts/game.js

```diff
@@ -78,19 +78,34 @@ export function getUniqueCard() {
   return chosenCard;
 } /* getUniqueCard */
 
+/**
+ * Reverses the oracle entry animation when the game has ended
+ */
+function reverseOracleAnimation() {
+  const oracle = document.querySelector('.oracle');
+  oracle.style.animationDirection = 'reverse';
+  oracle.style.animationFillMode = 'forwards';
+  oracle.classList.remove('oracle');
+  oracle.offsetHeight;
```

Comment:
Why is this `oracle.offsetHeight;` line here?

Instead of this mix of CSS and JS, why not just make a .reversed-oracle class that has these modifications? Then, you could just

```
oracle.classList.remove('forward-oracle');
oracle.classList.add('reverse-oracle');
```

Reply:
offsetHeight forces a reflow, otherwise the wizard just disappears when the reverse animation is triggered. As for the reversed oracle class, genuinely don't know why I didnt think of that lmao

Reply:
Hmm... I suppose the least hacky way to go about it would be to use the [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API), as then you can explicitly trigger it. But perhaps this is fine for now. @KristinEbu I'll defer to you